components/ui/Toggle.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ChangeEvent } from 'react'
+
+type Props = {
+  id: string
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void
+  isChecked: boolean
+  text?: string
+  isDisabled?: boolean
+}
+
+export const Toggle: React.FC<Props> = ({
```

Comment:
I'm not sure if the Toggle should look like this in dark mode:

<img width="421" alt="Screenshot 2024-08-06 at 13 21 34" src="https://github.com/user-attachments/assets/fab07ad2-fda1-4605-bbd0-7d8e4bbddfbb">

Reply:
what is your suggestion anyway?

Reply:
gray when off, color when on. how about it now?
<img width="276" alt="Screenshot 2024-08-06 at 11 26 19 at night" src="https://github.com/user-attachments/assets/51f627e6-7d77-4f6e-8b24-c4e94c1bc319">
<img width="251" alt="Screenshot 2024-08-06 at 11 27 30 at night" src="https://github.com/user-attachments/assets/be9c1ef0-c222-4313-b44d-67dd5da7896d">

Reply:
if theres a tooltip i'm fine with I debating if instead of the logo we should just say EOF or something I don't think anyone would identify the logo

Reply:
How about a dark background color in dark mode? Something that looks like the image below. 

<img width="109" alt="Screenshot 2024-08-06 at 19 31 45" src="https://github.com/user-attachments/assets/081ba7a8-4927-4d84-b98f-e73b75caefab">

Reply:
ok, I would keep the toggle as it is, and change the bg color on dark